src/_posts/databases/postgresql/2000-01-01-connecting.md

```diff
@@ -0,0 +1,139 @@
+
+
+## Understanding the Connection URI
+
+The connection URI is made of several **components** separated one from each
+other by a **delimiter**. For example, the `@` character is used to mark the
+end of the *userinfo* (credentials) component of the URI.
+
+In the case of PostgreSQL®, the connection URI provided by Scalingo is always
+formed as follows:
+
+```bash
+postgresql://[user]:[password]@[url]:[port]/[dbname]?sslmode=prefer
+```
+
+For more information about the connection URI syntax, please see
+[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) which defines the URI
+Generic Syntax.
+
+
+## Using the Connection URI
+
+**We strongly advise to use one of the two environment variables available to
+connect your application to your database**. Please don't use the value, but
+the environment variable itself.
+
+While the value of the provided connection URI **should not** change over time,
+**we don't guarantee it**. For example, the URI could change for maintenance
+reasons. In such a case, using the environment variable guarantees that your
+application can restart successfully without human intervention. Otherwise, you
+would have to update your code with the new value and trigger a new deployment,
+which would most probably contribute to a greater downtime.
+
+In most cases, you can pass the environment variable directly to the client
+library you are using. But sometimes, the library requires a specific URI
+format, invididual keypairs or another format. In such cases, you will probably
+need to parse the connection URI to retrieve the required values and build
+what's required by the library you are using. Our advice to use the environment
+variable still remains applicable.
+
+
+## Getting the Connection URI
+
+### Using the Dashboard
+
+1. From your web browser, open your [dashboard](https://dashboard.scalingo.com)
+2. Click on the application for which you want to get the connection URI
+3. Click on the **Environment** tab
+4. Locate the `SCALINGO_POSTGRESQL_URL` line and place your mouse over it to
+   unblur the value
+
+### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, list the environment variables available in your app
+   and filter the output using `grep`:
+   ```bash
+   scalingo --app my-app env | grep POSTGRESQL
+   ```
+   The output should look like this:
+   ```bash
+   DATABASE_URL=$SCALINGO_POSTGRESQL_URL
+   SCALINGO_POSTGRESQL_URL=postgresql://my_app_wxyz:ptojfrxzRi-lDfDYyahe@my-app-wxyz.postgresql.a.osc-fr1.scalingo-dbs.com:31000/my_app_wxyz?sslmode=prefer
+   ```
+
+
+## Enforcing TLS Connection
+
+All PostgreSQL® addons [support TLS](https://www.postgresql.org/docs/current/static/ssl-tcp.html)
+to encrypt all the network traffic between the client and the server.
+
+By default, all new PostgreSQL® databases have TLS **activated** (available),
```

Comment:
suggestion: I tend to think that "available" is clearer, I would only use this one

```suggestion
By default, all new PostgreSQL® databases have TLS **available**,
```